self-tests.yml

```diff
@@ -287,6 +287,8 @@ functions:
           JIRA_TOKEN_SECRET: ${jira_token_secret}
           JIRA_CONSUMER_KEY: ${jira_consumer_key}
           PAPERTRAIL_KEY_ID: ${papertrail_key_id}
+          AWS_KEY: ${self_tests_aws_key}
+          AWS_SECRET: ${self_tests_aws_secret}
```

Comment:
Instead of re-introducing static AWS credentials, can we instead add permissions to the assumed role in the self-tests? The infra team removed our AWS key permissions and swapped it out for [assuming a role](https://github.com/evergreen-ci/evergreen/blob/ca2fdbc875f2cb778d5f54b262eb5b0ec22b6fbb/self-tests.yml#L10) because it's better practice and support for static credentials in S3 operations will be removed in DEVPROD-5553.

Reply:
Absolutely! If there's already a role that exists during the self-tests I am happy to remove this and use the default identity for the tests